### PR TITLE
New effect: GTA 2077

### PR DESCRIPTION
**Pull request by juhana**
Forces most peds to t-pose. It also causes strange contortions together with some ped animations.

### CODE REVIEW

File: ConfigApp/Effects.cs [hunk 1]
```diff
@@ -548,7 +549,8 @@ public enum EffectType
             {EffectType.EFFECT_PLAYER_VR, new EffectInfo("Virtual Reality", EffectCategory.PLAYER, "player_vr", true, true)},
             {EffectType.EFFECT_HIGH_PITCH, new EffectInfo("High Pitch", EffectCategory.MISC, "misc_highpitch", true)},
             {EffectType.EFFECT_NO_SKY, new EffectInfo("No Sky", EffectCategory.MISC, "misc_nosky", true)},
-            {EffectType.EFFECT_PLAYER_GTA_2, new EffectInfo("GTA 2", EffectCategory.PLAYER, "player_gta_2", true, true)}
+            {EffectType.EFFECT_PLAYER_GTA_2, new EffectInfo("GTA 2", EffectCategory.PLAYER, "player_gta_2", true, true)},
```
Comment (ProfessorBiddle) [inline ConfigApp/Effects.cs:552]: Looks good to me, could lose this line though

**Review by ProfessorBiddle — APPROVED** [review]
Looks good, this will go along with Spinning Peds and Invisible vehicles really well

File: ConfigApp/Effects.cs (hunk 1 above)
Comment (juhana) [reply ConfigApp/Effects.cs:552]: Not sure which change you mean, the comma is needed as a separator for the new array element

File: ChaosMod/Effects/EffectsInfo.h [hunk 2]
```diff
@@ -527,5 +528,6 @@ const std::unordered_map<EffectType, EffectInfo> g_effectsMap =
 	{EFFECT_PLAYER_VR, {"Virtual Reality", "player_vr", true, {}, true}},
 	{EFFECT_HIGH_PITCH, {"High Pitch", "misc_highpitch", true, { EFFECT_GAMESPEED_X02, EFFECT_GAMESPEED_X05 }}},
 	{EFFECT_NO_SKY, {"No Sky", "misc_nosky", true}},
-	{EFFECT_PLAYER_GTA_2, {"GTA 2", "player_gta_2", true, { EFFECT_PLAYER_QUAKE_FOV, EFFECT_FLIP_CAMERA }, true}}
+	{EFFECT_PLAYER_GTA_2, {"GTA 2", "player_gta_2", true, { EFFECT_PLAYER_QUAKE_FOV, EFFECT_FLIP_CAMERA }, true}},
+	{EFFECT_PEDS_TPOSE, {"GTA 2077", "peds_tpose", true, { EFFECT_IN_THE_HOOD, EFFECT_SLIDY_PEDS }}}
```
Comment (Last0xygen) [inline ChaosMod/Effects/EffectsInfo.h:532]: „True“ has to be added to be a timed effect and short (see vr-effect above)

**Review by Last0xygen — CHANGES_REQUESTED** [review]

File: ChaosMod/Effects/EffectsInfo.h (hunk 2 above)
Comment (kolyaventuri) [inline ChaosMod/Effects/EffectsInfo.h:532]: Ditto Last0xygen. Otherwise looks good :+1:

**Review by kolyaventuri — CHANGES_REQUESTED** [review]

**Pull request merged**